docker-compose.yml

```diff
@@ -9,8 +9,6 @@
 # Database password: example
 # ADVANCED OPTIONS; Database host: postgres
 
-version: '3.1'
-
 services:
```

Comment:
We should also rename the file `compose.yaml` to be in sync with be Docker best practices.

Reply:
Can do, is there some documentation for this? This is a new thing to me.

Reply:
Yes: https://docs.docker.com/compose/compose-file/03-compose-file/

Just renaming the file should be enough.

Reply:
Hmm, for me any `docker compose command` will result with `no configuration file provided: not found` after renaming the file. Docker Compose version v2.22.0